index.html

```diff
@@ -1,84 +1,108 @@
+                constructor({ title }) {
+                  this.title = title;
+                  this.element = this._createInput();
                 }
-                getValue(){
-                  return this.input.value
+              
+                getValue() {
+                  return this.input.value;
                 }
+              
                 _createInput() {
-                const title = document.createElement("h4")
-                title.innerText = this.title
-                const input = document.createElement("input")
-                this.input = input
-                
-                const conteiner = document.createElement("div")
-                conteiner.className = "input"
-                
-                conteiner.appendChild(title)
-                conteiner.appendChild(input)
-                
-                return conteiner
+                  const title = document.createElement("h4");
+                  title.innerText = this.title;
+              
+                  const input = document.createElement("input");
+                  this.input = input;
+              
+                  const container = document.createElement("div");
+                  container.className = "input";
+              
+                  container.appendChild(title);
+                  container.appendChild(input);
+              
+                  return container;
                 }
-                }
-         </pre>
-     </section>
-     <section>
-         <h3>Experience:</h3>
-         <p>Tride to create web pages and and write code in JS for it.</p>
-         <a href="https://kiryuha94.github.io/test/" title="https://kiryuha94.github.io/test/" target=blank >My WEB</a>
-     </section>
-     <section>
-         <h3>Education:</h3>
-         <p>Only self-study</p>
-     </section>
-     <section>
-         <h3>English:</h3>
-         <p>A2. Now I pass an online course</p>
-     </section>
- </main>
- <footer>
-     <address>
-        Email: <a href="kirylpliushchenia94@gmail.com">kirylpliushchenia94@gmail.com</a>
-        Tel:<a href="tel:375298091021"> +375298091021</a>
-     </address>
- </footer>  
-</body>
+              }
+         </pre
+        >
+      </section>
+      <section class="experience">
+        <h3>Experience:</h3>
+        <p>Tride to create web pages and and write code in JS for it.</p>
+        <a
+          href="https://kiryuha94.github.io/test/"
+          title="https://kiryuha94.github.io/test/"
+          target="blank"
+          >My WEB</a
+        >
+      </section>
+      <section class="education">
+        <h3>Education:</h3>
+        <p>Only self-study</p>
+      </section>
+      <section>
+        <h3>English:</h3>
+        <p>A2. Now I pass an online course</p>
+      </section>
+    </main>
+    <footer>
+      <address>
+       <span><i class="fas fa-at"></i>
+        <a href="kirylpliushchenia94@gmail.com"
+          >kirylpliushchenia94@gmail.com</a>
```

Comment:
Why do you move > to the next line? What is your opinion: is it more readable than close a tag on the same line in this case?